features/old/api/features.feature

```diff
@@ -8,59 +8,60 @@ Feature: Features :)
     And an application plan "Basic" of provider "foo.3scale.localhost"
     Given current domain is the admin domain of provider "foo.3scale.localhost"
 
-  @javascript @wip
+  @javascript
   Scenario: Create a feature
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
     When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
+    Then I go to the edit page for plan "Basic"
     And I follow "New feature"
     And I fill in "Name" with "Free T-shirt"
+    And I fill in "System name" with "tee"
     And I fill in "Description" with "T-shirt with logo of our company for free."
-    And I press "Create Feature"
-    Then I should see enabled feature "Free T-shirt"
-    And provider "foo.3scale.localhost" should have feature "Free T-shirt"
-    And feature "Free T-shirt" should be enabled for plan "Basic"
+    And I press "Save"
+    Then I should see "T-shirt with logo of our company for free."
 
   # TODO: Test that the "No features yet" notice dissapears when first feature is created,
```

Comment:
I think it would be nice to implement this TODO, as we are on this test.

For reference, here is how it looks like:
![Screenshot from 2023-07-31 14-47-52](https://github.com/3scale/porta/assets/1270328/17943aee-d75e-4dc1-aeb8-cafef808f39b)

Reply:
Added